modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/dataSets.spec.ts

```diff
@@ -200,35 +201,55 @@ test('Create parameterized data set via UI', async ({dataSetsPage, page}) => {
 	});
 });
 
-test('Create data set via API', async ({
```

Comment:
I removed this test, as we are using API creation in many other places. This just needlessly uses CI resources.